pkg/kubelet/kuberuntime/kuberuntime_container_test.go

```diff
@@ -244,7 +244,7 @@ func TestGenerateContainerConfig(t *testing.T) {
 					Command:         []string{"testCommand"},
 					WorkingDir:      "testWorkingDir",
 					SecurityContext: &v1.SecurityContext{
-						RunAsNonRoot: &RunAsNonRoot,
+						RunAsNonRoot: &runAsNonRootTrue,
```

Comment:
Probably can be removed. @feiskyer 
But I don't want to turn this into a test cleanup PR. That can be done in a separate PR.